doc/source/authors.rst

```diff
@@ -108,6 +112,16 @@ and may not be the current affiliation of a contributor.
 26. Georgia Institute of Technology
 27. Emory University
 28. Harvard Medical School
+29. Institute for Advanced Simulation (IAS-6), Forschungszentrum Jülich, Germany
```

Comment:
@apdavison could you change my affiliation 23. to "Allen Institute for Neural Dynamics, Seattle, WA, United States"
?

Reply:
Is affiliation 23 incorrect (i.e. you never worked there, or didn't contribute to Neo while you worked there), or is it that you moved?

If it's just that you moved, then I would add the Allen Institute as a second affiliation.

Reply:
and I guess we should add CatalystNeuro as a third affiliation in that case?

Reply:
Hi @apdavison 

Yeah the institutution is correct, but I wonder whether we should keep all past institutions...What do you think?
IMO, it's better to keep the current institution(s), which for me are Allen Institute + CatalystNeuro

Reply:
Past institutions are potential copyright co-owners, so we need to keep track of them.

Reply:
If that's the case then I should probably add a second affiliation (I can do a PR myself soon). My MD/PhD will be from Harvard but my current lab is at Massachusetts General Hospital.